holoviews/core/data/dask.py

```diff
@@ -218,8 +218,6 @@ def groupby(cls, dataset, dimensions, container_type, group_type, **kwargs):
         for coord in indices:
             if any(isinstance(c, float) and np.isnan(c) for c in coord):
                 continue
-            if len(coord) == 1:
```

Comment:
Based on the filterwarnings, lets ignore this line when pandas is greater and equal than version 2.2